classes/payment/Payment.php

```diff
@@ -148,9 +148,9 @@ public function setContextId($contextId)
     }
 
     /**
-     * Set the type for this payment (PAYMENT_TYPE_...)
+     * Set the type for this payment (OJSPaymentManager::PAYMENT_TYPE_...)
      *
-     * @param int $type PAYMENT_TYPE_...
+     * @param int $type OJSPaymentManager::PAYMENT_TYPE_...
```

Comment:
Shouldn't be referring to OJS-specific classes from pkp-lib.

Reply:
Yep! I removed the prefix.